app/views/recipes/shopping_list.html.erb

```diff
@@ -0,0 +1,12 @@
+<h2>Shopping List</h2>
+
+<ul>
+<% @missing_ingredients.each do |ingredient| %>
+  <li>
+    <%= ingredient.name %> - <%= ingredient.quantity %> <%= ingredient.measurement_unit %>
+  </li>
+<% end %>
+</ul>
+
+<p>Total food items: <%= @missing_ingredients.count %></p>
+<p>Total price: <%= @total_price %></p>
```

Comment:
| @cancelei Shopping List |
|-|
| ![image](https://github.com/cancelei/recipe-in-rails/assets/93039359/f2dd9f0d-e355-433c-92dd-1062601d56aa) |
|Given Wireframe|
| ![image](https://github.com/cancelei/recipe-in-rails/assets/93039359/9aa7fdbf-6b8f-40fa-ad12-fe58d53c436e) |

- [ ] It would be nice if your shopping list should come on the correct route as in the project requirements which you can see in the given wireframe above :eyes: because right now your link is ` #green http://127.0.0.1:3000/recipes/1/generate_shopping_list` but according to the project requirements it should be `http://127.0.0.1:3000/generate_shopping_list`
- [ ] It would be nice If fix the major issue here which is that it doesn't show more than one recipe-foods because i have salt and garlic but it is only showing salt :fearful: Kindly fix this to make your project work as expected :blush:
- [ ] Kindly put all the missing details in the Shopping List which you can see in the given wireframe :+1: to make your design look similar to the given wireframe :blush:
- [ ] Great work on adding the shopping list but there are some issues you still need to fix to follow the requested changes by the previous reviewer :+1: The table should have column names and each item price is also missing Kindly make these changes to make your view look professional :+1: